controllers/cloudstackcluster_controller.go

```diff
@@ -176,13 +179,13 @@ func (reconciler *CloudStackClusterReconciler) SetupWithManager(mgr ctrl.Manager
 
 	// Add a watch on CAPI Cluster objects for unpause and ready events.
 	err = controller.Watch(
-		&source.Kind{Type: &capiv1.Cluster{}},
+		&source.Kind{Type: &clusterv1.Cluster{}},
 		handler.EnqueueRequestsFromMapFunc(
 			util.ClusterToInfrastructureMapFunc(infrav1.GroupVersion.WithKind("CloudStackCluster"))),
 		predicate.Funcs{
 			UpdateFunc: func(e event.UpdateEvent) bool {
-				oldCluster := e.ObjectOld.(*capiv1.Cluster)
-				newCluster := e.ObjectNew.(*capiv1.Cluster)
+				oldCluster := e.ObjectOld.(*clusterv1.Cluster)
```

Comment:
Reverting from the `capiv1` naming convention to further match what other providers are doing.